sphinx/writers/latex.py

```diff
@@ -275,6 +275,22 @@ def escape_abbr(text):
     return re.sub('\.(?=\s|$)', '.\\@', text)
 
 
+def rstdim_to_latexdim(width_str):
+    """Convert `width_str` with rst length to LaTeX length."""
+    match = re.match('^([0-9.]+)\s*(\S*)$', width_str)
```

Comment:
seems this regular expression allows ``1.2.3`` for example ? this would be wrong but is intercepted anyhow. (see other comment)